src/cdi_analytic/Analytic_Gray_Opacity.cc

```diff
@@ -33,7 +33,7 @@ namespace rtt_cdi_analytic {
 Analytic_Gray_Opacity::Analytic_Gray_Opacity(SP_Analytic_Model model_in,
                                              rtt_cdi::Reaction reaction_in,
                                              rtt_cdi::Model cdi_model_in)
-    : analytic_model(model_in), reaction(reaction_in), model(cdi_model_in) {
+    : analytic_model(std::move(model_in)), reaction(reaction_in), model(cdi_model_in) {
```

Comment:
I guess it's not clear why model_in can be moved but reaction_in and cdi_model_in are copied